tactics/hints.ml

```diff
@@ -1315,12 +1315,16 @@ let project_hint ~poly pri l2r r =
   let c = Reductionops.whd_beta sigma (mkApp (c, Context.Rel.to_extended_vect mkRel 0 sign)) in
   let c = it_mkLambda_or_LetIn
     (mkApp (p,[|mkArrow a Sorts.Relevant (lift 1 b);mkArrow b Sorts.Relevant (lift 1 a);c|])) sign in
-  let id =
+  let name =
```

Comment:
Strictly speaking this is indeed an id and it's the declare API using the wrong label. It doesn't matter much I guess.

Reply:
I had the same thought actually, indeed that's an inconsistency but then we usually speak of "the name of a constant" right? So seems to me that `name` may be a better label. I dunno, no problem in going back to `id`, but it seems more like a label for a UID / GUIID, etc....